dll/win32/wintrust/wintrust.spec

```
@@ -94,7 +94,7 @@
 @ stub WTHelperCheckCertUsage
 @ stub WTHelperGetAgencyInfo
 @ stdcall WTHelperGetFileHandle(ptr)
-#@ stub WTHelperGetFileHash
+@ stub WTHelperGetFileHash
```

```suggestion
@ stdcall -stub WTHelperGetFileHash(wstr long ptr ptr ptr ptr)
```
See
https://docs.microsoft.com/en-us/windows/win32/seccrypto/wthelpergetfilehash

---

No.
This is a stub that will silently fail, I prefer a stub that bombs out, so that we know it is actually used.

---

Alright. Then, why not add the parameters as a comment, as the line was before?